books/book.js

```diff
@@ -37,3 +37,42 @@ function displayBook(book) {
 
   main.innerHTML = currentBook;
 }
+
+var date = new Date();
+var dd = String(today.getDate()).padStart(2, '0');
+var mm = String(today.getMonth() + 1).padStart(2, '0');        //January is 0!
+var yyyy = today.getFullYear();
+
+var Price = document.getElementById("Price").value;
```

Comment:
the variable and the html element id should be lowercase, so `price `instead of `Price`